WORKSPACE

```diff
@@ -177,6 +177,14 @@ container_pull(
     #tag = "28",
 )
 
+# Pull nfs-server image
+container_pull(
+    name = "nfs-server",
+    digest = "sha256:2a8c1cad9156165d0be1ee2c78fb2342f8368193acff72eb276ff276512508fa",
+    registry = "index.docker.io",
+    repository = "slintes/nfs-ganesha",
```

Comment:
can we get a comment pointing to the repo this is built in just in case someone needs to update this container at some point.